mern-ecommerce/controllers/order.js

```diff
@@ -61,3 +61,15 @@ exports.updateOrderStatus = (req, res) => {
     }
   );
 };
+
+exports.purgeOrder = async (req, res) => {
+  try {
+    const result = await Order.findOneAndRemove({ _id: req.params.orderId });
+    if (!result) {
+      return res.status(404).json({ message: "Order not located" });
```

Comment:
The error message "Order not located" should be consistent throughout the application. It's better to define it as a constant in a separate file, e.g., `constants.js`, and import it where needed. This way, if the message needs to be changed, it can be done in one place.